get_files.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+from argparse import ArgumentParser
+
+from minio import Minio
+from tqdm import tqdm
+
+if __name__ == '__main__':
+    parser = ArgumentParser()
+    parser.add_argument('--secure', '-s', action="store_true",
```

Comment:
I agree. We needn't download all the files. So would we have args like `--bio` and `--epi`, or have the user specify a sub-folder path explicitly that they want to download? The safest bet is to just download everything, but if the number of files grows over time, it could be inconvenient for new users.